test-installer.sh

```
@@ -0,0 +1,45 @@
+
+echo "🧪 Testing httprunner installer scripts..."
+
+# Create a temporary directory for testing
+TEST_DIR=$(mktemp -d)
+echo "📁 Test directory: $TEST_DIR"
+
+# Copy installer scripts to test directory
+cp install.sh "$TEST_DIR/"
+cp install.ps1 "$TEST_DIR/"
+
+cd "$TEST_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)